helm/cluster-operator/values.yaml

```diff
@@ -13,3 +10,53 @@ pod:
 project:
   branch: "[[ .Branch ]]"
   commit: "[[ .SHA ]]"
+
+registry:
+  domain: docker.io
+  mirrors:
+  - giantswarm.azurecr.io
+  pullSecret:
+    dockerConfigJSON: ""
+
+  dockerhub:
+    token: token
+
+cni:
+  mask: 16
+  subnet: 10.1.0.0/16
+
+kubernetes:
+  api:
+    clusterAPIRange: 172.31.0.0/16
+  clusterDomain: cluster.local
+
+provider:
+  kind: ""
+
+release:
+  app:
+    config:
+      default: |
+        catalog: default
+        namespace: kube-system
+        useUpgradeForce: true
+      override: |
+        cert-exporter:
+          chart: "cert-exporter"
+        # chart-operator must be installed first so the chart CRD is
+        # created in the tenant cluster.
+        chart-operator:
+          chart:     "chart-operator"
+          namespace: "giantswarm"
+        # Upgrade force is disabled to avoid affecting customer workloads.
+        coredns:
+          useUpgradeForce: false
+        net-exporter:
+          chart: "net-exporter"
+        # Upgrade force is disabled to avoid affecting customer workloads.
+        nginx-ingress-controller:
+          useUpgradeForce: false
```

Comment:
Was that configured in installations before?

Reply:
yes, I took it from installations